src/nodedef.cpp

```diff
@@ -682,6 +682,8 @@ void ContentFeatures::updateTextures(ITextureSource *tsrc, IShaderSource *shdsrc
 	switch (drawtype) {
 	default:
 	case NDT_NORMAL:
+		material_type = (alpha == 255) ?
```

Comment:
I admit that I was not sure it that was necessary, I was just considering some mod using `use_texture_alpha = true` in their nodedef. Can't regular nodes use that property?